src/indexer.h

```diff
@@ -8,12 +8,28 @@
 #include "util/arr.h"
 // Preprocessors can store field data to this location
 typedef struct FieldIndexerData {
-  double numeric;  // i.e. the numeric value of the field
-  const char *geoSlon;
-  const char *geoSlat;
-  char **tags;
-  const void *vector;
-  size_t vecLen;
+  int isMulti;
```

Comment:
Don't we need to know that type here if we change it into a union? How the code knows to look at it as numeric or as tags?

Reply:
The context in which we handle it is enough (numericPreprocessor, fulltextPreprocessor or tagPreprocessor)

Reply:
So on which context we use isMulti? Can we squash it into the union?

Reply:
The numeric preprocessor is setting `isMulti` to `1` when handling `FLD_VAR_T_ARRAY`.
We need to distinguish between union members `numeric` (single) and `arrNumeric` (multi)
And soon also between `geoSlon`+`geoSlat` (single) and `arrGeoSlon`+`arrGeoSlat` (multi)
So we need the `isMulti` outside of the union.